core/templates/pages/exploration-player-page/services/content-translation-manager.service.ts

```diff
@@ -246,9 +246,13 @@ export class ContentTranslationManagerService {
       }
     }
 
+    // TODO(#14340): Remove the usage of jQuery.
     const element = $(card.getInteractionHtml());
     this.extensionTagAssemblerService.formatCustomizationArgAttrs(
       element, caValues);
+    // This throws "Object is possibly 'undefined'.". We need to suppress this
```

Comment:
Well, technically, you don't really know that, right? Probably need to assert somehow that `element` isn't undefined.

Ditto elsewhere.

Reply:
Done. I have added optional chaining.